src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -96,11 +94,7 @@ public static DueDate parseDate(String dueDate) throws ParseException {
         if (!DueDate.isValidDueDate(trimmedDueDate)) {
             throw new ParseException(DueDate.MESSAGE_CONSTRAINTS_DUE_DATE);
```

Comment:
Can you do a test and see how does it look when this message appears in the box, I have a feeling it might not show everything